core/store/migrate/migrations/00064_ensure_chain_specific_gas_limit.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
```

Comment:
The migration file name is different from our other migrations with the 2 leading zeroes instead of 3. I assume the goose generator created this. I don't think this is an issue but something to note and perhaps standardise to get everyone to use the generator when creating migrations

Reply:
We ought to standardise on 3 zeroes since this is what the generator produces.

Reply:
https://app.shortcut.com/chainlinklabs/story/16808/fix-goose-user-experience

Reply:
(renamed to two zeroes for this PR)